### ISSUE TITLE
Change order of notifications in material design

### ISSUE THREAD

**Issue by thibaultmol** [loomchild/sojourner-web#129]
I would like to note though that this implementation of the material design notifications at the bottom; have the problem that newer notifications don't go over it. Like if I logout accidentally and then log back in. the login message won't appear until the 'successful logout' notification has gone away. I think it might be better if it were to show the 'successful login' above the 'logout'. This is an example. But I meant that newer notifications should display over old notifications

**Comment by loomchild** [comment]
Yes, it makes sense and I am aware of this issue. I'll need to upgrade to the modern version of dependencies (Vuetify).
